lib/registration_util.dart

```diff
@@ -11,7 +10,10 @@ class RegistrationUtil{
   /// ...the password contains less than 2 digits
   /// ...the confirmed password is not the same as the password
 
-  bool validateRegistrationUserInput(String username, String password, String confirmedPassword){
-    return true;
+  bool validateRegistrationUserInput(
+      String username, String password, String confirmedPassword) {
+    return username.isNotEmpty &&
+        password.length >= 2 &&
+        confirmedPassword == password;
```

Comment:
A nghĩ nên dùng isNotBlank thay cho isNotEmpty.
// password contains less than 2 digits: What is a digit?